client/src/components/SessionLock.vue

```diff
@@ -0,0 +1,74 @@
+<script>
+import { mapState, mapMutations } from 'vuex';
+
+export default {
+  name: 'SessionLock',
+  inject: ['user', 'djangoRest'],
+  computed: {
+    ...mapState(['mainSession']),
+  },
+  methods: {
+    ...mapMutations(['setMainSession']),
+    async acquireLock() {
+      try {
+        this.djangoRest.acquireSession(this.mainSession.id);
+        this.setMainSession({ ...this.mainSession, lock_owner: this.user });
+      } catch (err) {
+        console.log(err);
+      }
+    },
+    async releaseLock() {
+      try {
+        this.djangoRest.releaseSession(this.mainSession.id);
+        this.setMainSession({ ...this.mainSession, lock_owner: null });
+      } catch (err) {
+        console.log(err);
```

Comment:
This has the effect of ignoring this error. It's generally preferable, in the case of behaviorally unhandled exceptions, to let them bubble up out of the stack. That will log them to the console anyway, and if we end up using Sentry or some other similar tool, we'll get all the nice features of telemetry, logging, and alerting.